app/src/components/modal/modal_your_connection/index.tsx

```diff
@@ -103,18 +104,30 @@ const AccountInfoAddress = styled.p`
   font-size: ${props => props.theme.fonts.defaultSize};
   color: ${props => props.theme.colors.textColorDark};
   margin: 0;
+  width: 78px;
+  line-height: ${props => props.theme.fonts.defaultLineHeight}
+  letter-spacing: 0.2px;
+  height: 16px;
 `
 
 const AccountInfoWallet = styled.p`
   font-size: 12px;
   color: ${props => props.theme.colors.textColorLighter};
   margin: 0;
+  line-height: 14px;
+  height: 14px;
```

Comment:
Same here...Next time try deleting height and see if its the same don't copy figma...